tests/topotests/bgp-evpn-overlay-index-gateway/PE1/bgp_vni_routes_base.json

```diff
@@ -0,0 +1,192 @@
+{
```

Comment:
topotests that cache json output and comp are not reviewable for all practical purposes; as we don't know what exactly the test is trying to validate. I know there are a lot of such tests; but it is not the best precedent IMO.

Reply:
We can discuss this in the meeting. Thanks!